python/qupled/quantum.py

```diff
@@ -280,3 +280,129 @@ def setGuess(self, fileName : str) -> None:
         guess.matsubara = hdfData["matsubara"]
         self.inputs.guess = guess
         
+class QVSStls(qc.VSStls, Qstls):
+
+    """
+    Class used to setup and solve the quantum VS-STLS scheme as described by
```

Comment:
Update description

Reply:
Not sure how to update it exactly. This is kinda tied to your next comment. There isn't much literature around qVS.